babel.cfg

```diff
@@ -0,0 +1,4 @@
+[python: navigator_engine/**.py]
+
+[extractors]
+bdg = navigator_engine.common.bdg_message_extractor:extract_bdg
```

Comment:
I didn't think we were using this babel extractor any more?

Reply:
Custom extractor has been removed